reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux'
+import { currentFlies } from './currentFlies'
+
+const rootReducer = combineReducers({
+	currentFlies,
```

Comment:
I don't imagine our store will have much, but this will still be a much more manageable way to handle our global state management.